modules/@apostrophecms/event-page/views/page.html

```diff
@@ -1,11 +0,0 @@
-{% extends data.outerLayout %}
```

Comment:
This was changed to index.html. page.html works to some degree, but I don't know if it includes all index page features.